### DISCUSSION TITLE
Appwrite database

### DISCUSSION THREAD

**Discussion by Bharathgit5** [community/community#52739] [Programming Help]
### Select Topic Area

Question

### Body

Iam new to appwrite and reactjs iam building a project which I want to store the user inputs from different components of my reactjs project . In one component I have created a document It was storing properly but I also want to store the user input from another component and it should store under the same document ud

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Bharathgit5 ! 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
